data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -18,7 +20,10 @@ MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE=$(image_for kube-client-agent)
 MACHINE_CONFIG_INFRA_IMAGE=$(image_for pod)
 
 KUBE_ETCD_SIGNER_SERVER_IMAGE=$(image_for kube-etcd-signer-server)
-
+if ! CLUSTER_ETCD_OPERATOR_IMAGE=$(image_for cluster-etcd-operator 2> /dev/null)
+then
+	CLUSTER_ETCD_OPERATOR_IMAGE=
+fi
```

Comment:
why the `2>`

Reply:
to suppress STDERR if the image does not exist. the result is still false I can remove if you prefer.

Reply:
Might fail with set e here.

Reply:
addressed